src/tiledb/cloud/geospatial/ingestion.py

```diff
@@ -1402,6 +1412,30 @@ def consolidate_meta(
     logger.info("max memory usage: %.3f GiB", max_memory_usage() / (1 << 30))
 
 
+def remove_dataset_type_from_array_meta(
+    dataset_uri: str,
+    *,
+    verbose: bool = False,
+):
+    """
+    Removes `dataset_type` meta if the ingested result is an array.
+    FIXME: This exists to fix an internal UI issue until formally fixed.
+
+    :param dataset_uri: dataset URI
+    :param verbose: verbose logging, defaults to False
+    """
+    logger = get_logger_wrapper(verbose)
+    if tiledb.object_type(dataset_uri) == "array":
+        logger.info(
+            "Ingested result is an array. " "Removing dataset_type from metadata..."
+        )
+        with tiledb.open(dataset_uri, mode="w") as array:
+            try:
+                del array.meta["dataset_type"]
+            except KeyError as exc:
+                logger.debug(f"Failed: {str(exc)}")
```

Comment:
```suggestion
                logger.debug(f"Failed to remove `dataset_type` key: {str(exc)}")
```

Reply:
Maybe info level so that it accompanies the previous message in a log file?